pytrip/cube.py

```diff
@@ -918,3 +918,13 @@ def write_trip_data(self, path):
             cube = cube.byteswap()
         cube.tofile(path)
         return
+
+    @classmethod
+    def save(cls, path):
+        """ Saves the object as a pickle object.
+        :params: path where file is stored.
+        """
+
+        import pickle
+        with open(path, 'wb') as f:
+            pickle.dump(cls, f, pickle.HIGHEST_PROTOCOL)
```

Comment:
Compability is not essential now. For me it is sufficient to add that as a future feature.